enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/util/BadgeList/BadgeList.tsx

```diff
@@ -21,7 +21,7 @@ export const BadgeList = <T,>({
   onRemoveItem,
   addButtonLabel,
 }: BadgeListProps<T>) => (
-  <Paper p="md" w="30rem">
+  <Paper p="md" w="30rem" bg="bg-white">
```

Comment:
Fixes the popovers always having a white background on dark-themed apps. Looks good now.

![CleanShot 2568-01-28 at 00 09 56@2x](https://github.com/user-attachments/assets/e3c2adef-a5a8-4a6f-a539-f5418f396d1a)

Reply:
why does `bg-white` work here? Isn't it supposed to be white? How does this work?

Reply:
@WiNloSt This is because `colors.background` maps to `bg-white`, and we do use `bg-white` in many places in the current codebase. Here is the corresponding mapping:

https://github.com/metabase/metabase/blob/c345c1c79202173ee10b36b73263ca27c291c0a6/frontend/src/metabase/embedding-sdk/theme/embedding-color-palette.ts#L51-L55

Reply:
would it make more sense to use `var(--mb-color-background)` instead then? `background` maps to 2 colors, the first one isn't a semantic color. This can be applied to other places that use `bg-white` as well. I'm cool with keeping the mapping to support old components, but don't you think we should should  semantic colors for newer code?

Reply:
@WiNloSt ah, good point! I always use CSS variables when in CSS modules, but I sometimes forget to use them with Mantine style props. Will keep this in mind the next time then